Sprint-2/1-key-errors/1.js

```diff
@@ -1,20 +1,26 @@
 // Predict and explain first...
-
+// changes decimal numbers to percentage
 // Why will an error occur when this program runs?
 // =============> write your prediction here
 
 // Try playing computer with the example to work out what is going on
 
-function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
-  const percentage = `${decimalNumber * 100}%`;
+// function convertToPercentage(decimalNumber) {
+//   const percentage = `${decimalNumber * 100}%`;
 
-  return percentage;
-}
+//   return percentage;
+// }
 
-console.log(decimalNumber);
+// console.log(decimalNumber);
 
 // =============> write your explanation here
-
+// trying to redeclare the variable called decimalNumber twice in the function.
```

Comment:
also- the code was not invoking the `convertToPercentage` function which you corrected